.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,124 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
```

Comment:
This seems unrelated to the scope of this PR. If you want to suggest using Github workflows over Travis, please make a separate PR.